kubernetes-manifests/lanebreach-frontend.deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: node-lanebreach-frontend
+  labels:
+    app: node-lanebreach
+    tier: frontend
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: node-lanebreach
+        tier: frontend
+    spec:
+      containers:
+      - name: frontend
+        image: node-lanebreach-frontend
+        imagePullPolicy: IfNotPresent
```

Comment:
We probably want the pull policy to be `always` for the front end, right?

Reply:
Yes, good catch, I had swapped that for some local network when testing. Forgot to switch back